src/App.js

```diff
@@ -73,6 +85,7 @@ export default function App() {
             address={features.properties.address}
             liveDate={features.properties.live_date}
             precinct={features.properties.precinct}
+            handleFlyTo={() => handleFlyTo(features.geometry.coordinates)}
```

Comment:
Here applies the same principal as above:

       ` {participantData.map(participant => ...}`